typegraph/python/typegraph/wit.py

```diff
@@ -9,6 +9,11 @@
 from typegraph.gen.exports.runtimes import Runtimes
 from typegraph.gen.exports.utils import Utils
 from typegraph.host.host import HostImpl
+from typegraph.gen.exports.core import (
+    ArtifactResolutionConfig,  # noqa
+    MigrationConfig,  # noqa
```

Comment:
why are those required? maybe add a short explanation after #noqa:

Reply:
It's just a simple indirection to provide import similar to the node implementation.